packages/material-ui/src/NativeSelect/NativeSelect.js

```diff
@@ -22,7 +22,7 @@ export const styles = theme => ({
     userSelect: 'none',
     paddingRight: 32,
     borderRadius: 0, // Reset
-    width: 'calc(100% - 32px)',
+    width: 'calc(100% - 32px) !important',
```

Comment:
We should avoid the usage of `!important` as much as possible. This hide something else.

Reply:
Changing the order of the styling is another option. Do you want me to change it accordingly?